index.html

```diff
@@ -147,12 +156,31 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
           purpose is to be handed to {{BrowserCaptureMediaStreamTrack/cropTo}} as input.
         </p>
         <pre class="idl">
-          [Exposed = Window]
+          [Exposed = Window, Serializable]
           interface CropTarget {
             // Intentionally empty; just an opaque identifier.
           };
         </pre>
         <dl data-link-for="CropTarget" data-dfn-for="CropTarget"></dl>
+
+        <p id="serializable-croptarget">
+          {{CropTarget}} objects are [= serializable objects =] [[!HTML]]. Their [= serialization
+          steps =], given <var>value</var> and <var>serialized</var>, are:
+        </p>
+        <ol>
+          <li>Make <var>serialized</var> a randomly generated, globally unique {{DOMString}}.</li>
```

Comment:
Internal slot makes sense to me. Could you suggest the exact wording?

Reply:
You cannot set _serialized_ to a value. You need to set _serialized_.[[Data]] or some such to a value. (Otherwise you also overwrite _serialized_.[[Type]] and break the caller. This should probably be documented better, maybe file an issue against whatwg/html?

I don't really have enough context here to provide further suggestions about what might need to be done.

Reply:
Added [[Data]].

Reply:
@jan-ivar, the current text allows the user agent to determine its own implementation. A mapping is one possibility.